src/Controller/TestController.php

```diff
@@ -1,23 +1,21 @@
 <?php
```

Comment:
I would personally prefer a built-in connection check at admin settings form load, utilizing the Data Api. Marking the controller deprecated for now sound good. Done @ 288b9fe53a171da2090858252877b5c8e4864eef.